src/core/bilibili.ts

```diff
@@ -138,18 +138,18 @@ const checkUrl = (url: string) => {
 }
 
 // 检查url是否有重定向
-const checkUrlRedirect = async (videoUrl: string) => {
+const checkUrlRedirect = async (videoUrl: string, videoType: string = '') => {
```

Comment:
remove the ' = ''' works for me, otherwise it will throw a compilation error

Reply:
It turns out that I "cannot include default parameters in function type definitions".  
Lesson learned.